src/Tinfoil/Digest.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+module Tinfoil.Digest(
```

Comment:
Encode rather than Digest?

More names, take with a grain of salt (and I know these were already there), just throwing things out there and maybe we can discuss at once a bunch of these things.